convert/convert.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ * Conversion between different character encodings
+ *
+ * @authors
+ * Copyright (C) 1999-2002,2007 Thomas Roessler <roessler@does-not-exist.org>
+ *
+ * @copyright
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
Add a `@page lib_convert` here (cut'n'paste from other `lib.h`)

You'll also need to as `lib_convert` to the list of libraries in `main.c`

Reply:
If there is only one file `lib.c` in the lib, do I omit the file table under `@page`?

Reply:
I keep meaning to docment this...

- Library XYZ
- `main.c` -- `@ref lib_xyz`
- `xyz/*.c` -- `@page xyz_foo`, `@page xyz_bar`, ...
- `xyz/lib.h` -- table `| xyz/foo.c | @subpage xyz_foo`, ... (**Note**: **sub**page)

That links everything up, so Doxygen can generate the ["Libraries" tab](https://neomutt.org/code/).